pkg/apis/build/v1alpha2/build_pod_test.go

```diff
@@ -654,6 +656,11 @@ func testBuildPod(t *testing.T, when spec.G, it spec.S) {
 					Value: build.Spec.Source.Git.Revision,
 				},
 			)
+			assert.Contains(t, pod.Spec.InitContainers[0].Env,
+				corev1.EnvVar{
+					Name:  "GIT_INITIALIZE_SUBMODULES",
+					Value: strconv.FormatBool(build.Spec.Source.Git.InitializeSubmodules),
```

Comment:
Not sure if it's intentional, but you imported a third party library `cuelang.org/go/pkg/strconv` and not the std lib `strconv`. Either way, `fmt.Sprintf` also does the same job

Reply:
whoops did not mean to import that lol